pkg/minikube/cruntime/cruntime.go

```diff
@@ -167,11 +167,14 @@ func disableOthers(me Manager, cr CommandRunner) error {
 // enableIPForwarding configures IP forwarding, which is handled normally by Docker
 // Context: https://github.com/kubernetes/kubeadm/issues/1062
 func enableIPForwarding(cr CommandRunner) error {
-	c := exec.Command("sudo", "modprobe", "br_netfilter")
-	if _, err := cr.RunCmd(c); err != nil {
-		return errors.Wrap(err, "br_netfilter")
+	c := exec.Command("sudo", "sysctl", "net.netfilter.nf_conntrack_count")
+	if rr, err := cr.RunCmd(c); err != nil {
+		glog.Infof("couldn't verify netfilter by %q which might be okay. output=%q error=%v", rr.Command(), rr.Output(), err)
```

Comment:
I believe rr.Output() here is redundant: RunCmd errors already include stdout/stderr, or at least non-kic drivers do. Can we fix kic's RunCmd to be consistent instead?

Reply:
sure thing ! I

Reply:
I just noticed we have same thing in kic as well.
```
fmt.Errorf("%s: %v\nstdout:\n%s\nstderr:\n%s", rr.Command(), err, rr.Stdout.String(), rr.Stderr.String())```